src/live_restore/live_restore_fs.c

```diff
@@ -926,8 +933,6 @@ __wti_live_restore_cleanup_stop_files(WT_SESSION_IMPL *session)
     WT_DECL_ITEM(buf);
     WT_DECL_ITEM(filepath);
 
-    fs->finished = true;
```

Comment:
This field is redundant with the global live restore state. We instead check that we're past the `WTI_LIVE_RESTORE_STATE_BACKGROUND_MIGRATION` stage